src/utils/index.ts

```diff
@@ -242,18 +242,16 @@ export const print = (x: string) => Deno.stdout.write(encoder.encode(`${x}\n`))
 
 
 ///////////////////////////////////////////////////////////////////////// misc
-export function panic(message?: string): never {
-  throw new Error(message)
-}
+import TeaError, { UsageError, panic } from "./error.ts"
+export { TeaError, UsageError, panic }
```

Comment:
export `wrap` to clean up the other import lines? (useDownload.ts and useInventory.ts)

Reply:
I didn't want to export wrap generally as the name is too generic. So wanted it namespaced. I guess we can import * as error and export that?

Reply:
exported as `error` and imports fixed